warp10/src/main/java/io/warp10/continuum/Configuration.java

```diff
@@ -1729,7 +1739,12 @@
    * Port onto which the egress server should listen
    */
   public static final String EGRESS_PORT = "egress.port";
-  
+
+  /**
+   * Port onto which the egress server should listen
+   */
+  public static final String EGRESS_TCP_BACKLOG = "egress.tcp.backlog";
```

Comment:
Needs to be split in something like egress.http.backlog and egress.ssl.backlog to add the same logic to the SSL endpoint.